client-example.sh

```diff
@@ -76,6 +75,7 @@ fi
 
 if [ $STATUS -eq 404 ];
 then
-    rm -f $CSRSUM
-    curl ${CURL_OPTS} --data-binary @$MY_CSR ${POST_URL}/${CSRSUM}
+  rm -f $1/$CSRSUM
```

Comment:
inconsistent quoting with  above statements.  

Also, never a good idea to refer to $1 more than once in a single script. If it's worth using, give it a name.

SUBDIR="$1"  or so.